### PR TITLE
Update project ppl page with latest changes

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | Update project ppl page with latest changes: QA alumni, Product alumni, change in roles and a global update of team ppl in general
| Fixed ticket |

### CODE REVIEW

File: content/project-organization/people-and-roles.md [hunk 1]
```diff
@@ -98,47 +103,28 @@ Alumni:
...
+- Ibrahima Sow ([@sowbiba](https://github.com/sowbiba))
+- Louis Authie ([@LouisAUTHIE](https://github.com/LouisAUTHIE))
 - Salim Benouamer ([@lartist](https://github.com/lartist))
 - Thomas Baccelli ([@atomiix](https://github.com/atomiix))
 
 ## Quality Council
 
 The quality council provides the quality assurance vision for the project and has authority on related subjects, such as verification of pull requests, test automation strategy, bug triage process, and quality assessment.
 
-### Issue Managers
```
Comment (matks) [inline content/project-organization/people-and-roles.md:108]: Removed this team with approval of @Robin-Fischer-PS as it was duplicate of "Software Testers" team

**Review by Robin-Fischer-PS — APPROVED** [review]

File: content/project-organization/people-and-roles.md [hunk 2]
```diff
@@ -29,11 +28,10 @@ The product council is in charge of product vision for the project, ensuring tha
 
 Current members:
 
-- Fabien Vallon ([@fabienvallon](https://github.com/fabienvallon)) - **Council Lead**
+- Amine Ouanes ([@amine-O](https://github.com/amine-O)) - **Council Lead**
 - Daniel Hlavacek ([@Hlavtox](https://github.com/Hlavtox))
 - Jean-François Viguier ([@jf-viguier](https://github.com/jf-viguier))
```
Comment (kpodemski) [inline content/project-organization/people-and-roles.md:33]:
```suggestion
- Daniel Hlavacek ([@Hlavtox](https://github.com/Hlavtox)) - external contributor not affiliated with PrestaShop SA 
- Jean-François Viguier ([@jf-viguier](https://github.com/jf-viguier)) - external contributor not affiliated with PrestaShop SA 
```

@matks wdyt?

Comment (matks) [reply content/project-organization/people-and-roles.md:33]: No opinion on this topic, so okay for me

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thx @matks

**Pull request merged**